assets/content/footer.html

```diff
@@ -108,26 +108,26 @@ <h5 class="heading">About</h5>
                 <div class="col-md-6">
                     <ul class="nav nav-footer justify-content-end">
                         <li class="nav-item">
-                            <a href="https://sefglobal.org/" class="nav-link">Home</a>
+                            <a href="../index.html" class="nav-link">Home</a>
                         </li>
                         <li class="nav-item">
-                            <a href="https://sefglobal.org/about.html" class="nav-link">About</a>
+                            <a href="../about/about.html" class="nav-link">About</a>
                         </li>
                         <li class="nav-item">
                             <a href="https://www.youtube.com/channel/UClw7QbeW_FOJz_fDMvjXsJw/" class="nav-link"
                                target="_blank">Portfolio</a>
                         </li>
                         <li class="nav-item">
-                            <a href="https://medium.com/@SustainableEduF" class="nav-link" target="_blank">Blog</a>
```

Comment:
@yohanym95 this blog link you've updated is yet to have any content right ? i suggest leaving it to redirect to medium until the content is updated.

@jayasanka-sack